core/iwasm/common/wasm_shared_memory.c

```diff
@@ -119,7 +119,9 @@ notify_stale_threads_on_exception(WASMModuleInstanceCommon *module_inst)
                          (void *)&total_elem_count);
 
     /* allocate memory */
-    if (!(args.addr = wasm_runtime_malloc(sizeof(void *) * total_elem_count))) {
+    const uint32 total_elem_count_size = sizeof(void *) * total_elem_count;
+    if (total_elem_count_size
```

Comment:
should we just return from this function if `total_elem_count` is zero?

Reply:
done